docs/how-to/rocm-for-hpc/index.rst

```diff
@@ -0,0 +1,239 @@
+          other Earth-system simulation components for use in climate, regional
+          climate, and weather studies.
+
+      * - Benchmark
+        - `HPCG <https://github.com/amd/InfinityHub-CI/tree/main/hpcg>`_
+        - High Performance Conjugate Gradient Benchmark: a complement to the
+          High Performance LINPACK (HPL) benchmark. The computational and data
+          access patterns of HPCG are designed to closely match a broad set of
+          important applications not represented by HPL, and to incentivize
+          computer system designers to invest in capabilities that benefit the
+          collective performance of these applications.
+
+      * -
+        - `rocHPL <https://github.com/amd/InfinityHub-CI/tree/main/rochpl>`_
+        - Implementation of the High Performance LINPACK (HPL) benchmark on the
+          ROCm platform: a benchmark which solves a uniformly random system of
+          linear equations and reports floating-point execution rate.
+
+      * -
+        - `rocHPL-MxP <https://github.com/amd/InfinityHub-CI/tree/main/hpl-mxp>`_
+        - Benchmark that highlights the convergence of HPC and AI workloads by
+          solving a system of linear equations using novel, mixed-precision
+          algorithms.
+
+      * - Tools and libraries
+        - `ROCm with GPU-aware MPI container <https://github.com/amd/InfinityHub-CI/tree/main/base-gpu-mpi-rocm-docker>`_
+        - Base container for GPU-aware MPI with ROCm for HPC applications. This
+          project provides boilerplate for building and running a Docker
+          container with ROCm supporting GPU-aware MPI implementations using
+          either OpenMPI or UCX.
+
+      * -
+        - `Kokkos <https://github.com/amd/InfinityHub-CI/tree/main/kokkos>`_
+        - C++ programming model for writing performant portable applications for
+          use across HPC platforms. It provides abstractions for both parallel
+          execution of code and data management. Kokkos targets complex node
+          architectures with N-level memory hierarchies and multiple types of
+          execution resources.
+
+      * -
+        - `PyFR <https://github.com/amd/InfinityHub-CI/tree/main/pyfr>`_
+        - Open-source Python framework for solving advection-diffusion type
+          problems on streaming architectures using the flux reconstruction
+          approach (Huynh). PyFR solves a range of governing systems on mixed
+          unstructured grids containing various element types. It's designed to
+          target a range of hardware platforms via an in-built domain-specific
+          language derived from the Mako templating engine.
+
+      * -
+        - `RAJA <https://github.com/amd/InfinityHub-CI/tree/main/raja>`_
+        - Library of C++ software abstractions that enables architecture and
+          programming model portability for HPC applications. RAJA is primarily
```

Comment:
Should this be "RAJA was primarily..."
The present tense sounds odd here but perhaps it's still being developed.